StereoKitC/tools/file_picker.cpp

```diff
@@ -53,13 +53,16 @@ namespace sk {
 struct fp_item_t {
 	char *name;
 	bool  file;
+	long size;
```

Comment:
I would prefer using some variant of `int64_t` or `int32_t` instead of `long` or `int` (throughout this PR)! `long` and `int` have some amount of ambiguity in C/C++.

Reply:
File sizes seem to cap out around 16TB, which I believe don't fit in just a 32 bit integer, so I believe this specifically should probably still be an `int64_t`